cypress/integration/userCanNavigateThroughApplication.feature.js

```diff
@@ -0,0 +1,8 @@
+describe("User can navigate the app", () => {
+  beforeEach(() => {
```

Comment:
The "before" hook is preferred, beforeEach is slower and it's used in particular tests, not needed for this simple test.